src/models/channel-parameters.ts

```diff
@@ -1,4 +1,10 @@
-import type { Collection } from './collection';
+import { Collection } from './collection';
 import type { ChannelParameterInterface } from './channel-parameter';
 
-export type ChannelParametersInterface = Collection<ChannelParameterInterface>
\ No newline at end of file
+export type ChannelParametersInterface = Collection<ChannelParameterInterface>;
+
+export class ChannelParameters extends Collection<ChannelParameterInterface> implements ChannelParametersInterface {
+  override get(id: string): ChannelParameterInterface | undefined {
+    return this.collections.find(parameter => parameter.id() === id);
+  }
+}
```

Comment:
You'll find a ton of these too. I found that these collection implementations are always the same so I'm moving them from v2 to the models' directory. Check out the first commit of this PR to see all the related changes.